src/pocketmine/item/Item.php

```diff
@@ -619,7 +620,6 @@ private static function initCreativeItems(){
 		self::addCreativeItem(Item::get(Item::GLASS, 0));
 		self::addCreativeItem(Item::get(Item::GLOWSTONE_BLOCK, 0));
 		self::addCreativeItem(Item::get(Item::VINES, 0));
-		self::addCreativeItem(Item::get(Item::NETHER_REACTOR, 0));
```

Comment:
What is this for? its not in the description

It looks like you are deleting the nether reactor? Why in this? You made lots of requests about this.